components/Input.js

```diff
@@ -0,0 +1,89 @@
+import { nanoid } from "nanoid";
+import styled from "styled-components";
+
+export default function Input({ onCreateEntry }) {
+  function handleSubmit(event) {
+    event.preventDefault();
+    const nameInput = event.target.name.value;
+    const ideaInput = event.target.idea.value;
+    const dateInput = event.target.date.value;
+    const newEntry = {
+      id: nanoid(),
+      name: nameInput,
+      birthday: dateInput,
+      ideas: [ideaInput],
```

Comment:
An array seems unnecessary (at least right now). There can never be more than one item.

Reply:
I had to make it an array because otherwise the join-method didn't work. But that was with dummy-data, maybe it's different after I added more features like editing etc.